components/widgets/CurrentData/patterns/Zenith/index.js

```diff
@@ -0,0 +1,67 @@
+import PropTypes from "prop-types";
+import { useTranslation } from "react-i18next";
+import { formatAngle } from "@/helpers/formatters";
+import * as Styled from "./styles";
+
+const Zenith = ({ zenith = 0, isCondensed = false, className }) => {
+  const { t } = useTranslation();
+  const width = 52;
+  const height = width;
+  const clamped = Math.max(Math.min(zenith, 90), 0);
```

Comment:
clamp the value we receive in case it's something whacky, we don't want to see the telescope pointed at the ground